examples/docs_snippets/docs_snippets/concepts/ops_jobs_graphs/dep_dsl.py

```diff
@@ -42,7 +42,7 @@ def construct_graph_with_yaml(yaml_file, op_defs) -> GraphDefinition:
         name=yaml_data["name"],
         description=yaml_data.get("description"),
         node_defs=op_defs,
-        dependencies=deps,
+        dependencies=deps,  # type: ignore[arg-type]
```

Comment:
It's preferable to avoid "type: ignore"s in user-facing docs snippets. Is there a way we can address this type issue?

Reply:
It won't be using facing - we strip it out as of #10138